sql/colflow: TestVectorizedFlowDeadlocksWhenSpilling failed

sql/colflow.TestVectorizedFlowDeadlocksWhenSpilling [failed](https://tanzanite.cluster.engflow.com/invocations/default/021ef808-9cb2-42e0-a127-e7a731538fde?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL2NvbGZsb3c6Y29sZmxvd190ZXN0) on release-24.2 @ [6494c4e0e13d6f805e8e58396279dae1fd2e6fd9](https://github.com/cockroachdb/cockroach/commits/6494c4e0e13d6f805e8e58396279dae1fd2e6fd9):


```
=== RUN   TestVectorizedFlowDeadlocksWhenSpilling
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestVectorizedFlowDeadlocksWhenSpilling1623599267
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    vectorized_flow_deadlock_test.go:71: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/sql/colflow_test/pkg/sql/colflow/vectorized_flow_deadlock_test.go:71
        	Error:      	Should be false
        	Test:       	TestVectorizedFlowDeadlocksWhenSpilling
        	Messages:   	pq: query execution canceled
    panic.go:626: -- test log scope end --
test logs left over in: outputs.zip/logTestVectorizedFlowDeadlocksWhenSpilling1623599267
--- FAIL: TestVectorizedFlowDeadlocksWhenSpilling (78.63s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestVectorizedFlowDeadlocksWhenSpilling.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-44892

--- a/pkg/sql/colflow/vectorized_flow_deadlock_test.go
+++ b/pkg/sql/colflow/vectorized_flow_deadlock_test.go
@@ -31,7 +31,7 @@ func TestVectorizedFlowDeadlocksWhenSpilling(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.UnderStress(t, "the query might take longer than timeout under stress making the test flaky")
+	skip.UnderDuress(t, "the query might take longer than timeout under duress making the test flaky")
 
 	vecFDsLimit := 8
 	envutil.TestSetEnv(t, "COCKROACH_VEC_MAX_OPEN_FDS", strconv.Itoa(vecFDsLimit))

